MediaAdmin/FileUtils/Image/ImagickImageManager.php

```diff
@@ -11,13 +11,19 @@
 class ImagickImageManager implements ImageManagerInterface
 {
     protected $imagickFactory;
+    protected $maxWidth;
+    protected $maxHeight;
 
     /**
      * @param ImagickFactory $imagickFactory
+     * @param int            $maxWidth
+     * @param int            $maxHeight
      */
-    public function __construct(ImagickFactory $imagickFactory)
+    public function __construct(ImagickFactory $imagickFactory, $maxWidth = 5000, $maxHeight = 5000)
```

Comment:
I don't understand why you add default values as you inject them in service definition.